src/server/dummy-data/promotions.json

```diff
@@ -0,0 +1,16 @@
+{
+  "slides": [
+    {
+      "id": 0,
+      "content": "/images/slidesImgs/female.png"
```

Comment:
`content` change to `img`

add `link` prop. 
Take a look footer links implementation from Andrii

Reply:
oh, i got it, changing rn